components/idp-mgt/org.wso2.carbon.idp.mgt/src/main/java/org/wso2/carbon/idp/mgt/IdentityProviderManager.java

```diff
@@ -206,6 +206,7 @@ public IdentityProvider getResidentIdP(String tenantDomain)
         if (StringUtils.isBlank(oauth2TokenEPUrl)) {
             oauth2TokenEPUrl = IdentityUtil.getServerURL(IdentityConstants.OAuth.TOKEN, true, false);
```

Comment:
Are we still using the previous util method as well? Can't we use the new ServiceURLBuilder hear as well?

Reply:
Yes. We can change these to use the ServiceURLBuilder.

Reply:
The PR is now updated.